providers/aws/test_helpers.go

```diff
@@ -0,0 +1,284 @@
+package aws
```

Comment:
Any reason why this file is called test_helpers.go? Not having a _test.go extension means this file will be compiled with (although probably not linked into) the actual virtual kubelet binary. helpers_test.go would be a better name.

Were you planning to call these functions from any other file than provider_test.go? If not, should we just move these functions to the end of provider_test.go?

Reply:
I can just include them, but was worried that the provider_test would quickly grow in size especially when adding other tests.

Reply:
Sounds good.

Reply:
Changed 👍